src/containers/PharmacyList/PharmacyList.test.tsx

```diff
@@ -4,13 +4,20 @@ import { VaccineAvailabilityExpandedResponse } from "../../apiClient";
 import { server } from "../../mocks/server";
 import { render, screen } from "../../testUtils";
 import { PharmacyList } from "./PharmacyList";
+import { format } from "date-fns-tz";
+
+const formattedDate = format(
```

Comment:
Awesome, now we'll get consistent results irrespective of the timezone of the machine the test is being run on

👍